chipsec/cfg/1022/strix.xml

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0"?>
+<configuration platform="Strix Point">
+<!--
+XML configuration for Strix Point based platforms
+
+Reference: PPR for AMD Family 17h Model 18h B1. 55570-B1 Rev 3.16 - Apr 14, 2021.
+
+The following registers are defined in the reference PPR but have not been included:
+- ACP: the location of the BAR for APC access is not defined in the reference.
+- SMM: unreachable from the kernel.
+- SBTSI: unclear how to access this address space.
+- PMC events: no support in chipsec for performance monitor counters.
+-->
+
+  <!-- #################################### -->
+  <!--                                      -->
+  <!-- Information                          -->
+  <!--                                      -->
+  <!-- #################################### -->
+  <info family="core">
+    <sku did="0x1507" name="Strix" code="Strix" longname="Strix Point Root Complex" />
```

Comment:
I used `did` to identify sku when initially porting AMD to chipsec, but I think it would be better to use `cpuid`.  

Unlike Intel, AMD doesn't identify architecture is terms of PCH device id, but based on family/model/revision. Eg, the Dali PPR is titled "AMD Family 17h Model 20h, Revision A1". This maps to a cpuid of 0820F01 (unless I got the math wrong). I'm not sure what the correct cpuid is for Strix because the PPR isn't public yet, but I'd suggest a line similar to:

    <sku cpuid="0x0820F01" name="dali" code="dali" longname="dali" />

Reply:
Currently we don't support adding cpuid to a sku tag, but I think this would be a great addition.